include/jobs.hrl

```diff
@@ -33,7 +33,9 @@
 		    | {config, file:name()}
                     | {group_rates, [{q_name(), [option()]}]}
 		    | {counters,    [{q_name(), [option()]}]}
-		    | {interval, integer()}.
+		    | {interval, integer()}
+            | {max_time, non_neg_integer()}
+            | {max_size, non_neg_integer()}.
```

Comment:
Both `max_time` and `max_size` can actually also be `undefined`, which is interpreted as "no upper limit".